src/test/RecipeCard.test.js

```diff
@@ -0,0 +1,128 @@
+import axios from 'axios'
+import React from 'react'
+import {
+    render,
+    screen,
+    waitFor,
+    fireEvent,
+    cleanup,
+} from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import RecipeCard from '../RecipeCard'
+import { BrowserRouter } from 'react-router-dom'
+
+jest.mock('axios')
```

Comment:
this is not a hard rule, but I used to mock just one level above the component behind tested, in this case I would have mocked the **api** module